app/src/main/AndroidManifest.xml

```diff
@@ -97,6 +101,18 @@
             android:name=".view.activity.DevicesListActivity"
             android:parentActivityName=".view.activity.DashboardActivity"
             android:screenOrientation="portrait"/>
+        <activity
+            android:name=".view.activity.SimpleAuthenticatorRegistrationActivity"
```

Comment:
I would change the word `simple` in all names to something else, it just doesn't sound right... I'm also missing the word `custom`.
My suggestions are:
`RudimentaryCustomAuthenticatorRegistrationActivity`
`SampleCustomAuthenticatorRegistrationActivity`
`CustomAuthenticatorRegistrationActivity` - the whole app is called `Example app`, so it is already given it's an example

Reply:
Partially fixed. Custom authenticator implementations have "custom" in name, because those are SDK-related classes. In other context (authenticator activity) the `custom` name is not really required. For example if you use Eyeverify, you can call the authenticator that implements `OneginiCustomAuthenticator` the `EyeverifyCustomAuthenticator`, but the eyeverify classes like Activity are just called `EyeverifyActivity`